x/ccv/consumer/module_test.go

```diff
@@ -48,9 +45,7 @@ type ConsumerTestSuite struct {
 }
 
 func (suite *ConsumerTestSuite) SetupTest() {
-	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 2)
-	suite.providerChain = suite.coordinator.GetChain(ibctesting.GetChainID(1))
-	suite.consumerChain = suite.coordinator.GetChain(ibctesting.GetChainID(2))
+	suite.coordinator, suite.providerChain, suite.consumerChain = simapp.NewProviderConsumerCoordinator(suite.T())
```

Comment:
@sainoe I was wondering about this change in particular. There are three more similar instances (I will tag you).

I will also tag you in the `simapp.NewProviderConsumerCoordinator` code